public_html/wp-content/plugins/camptix-attendance/addons/attendance-ui.php

```diff
@@ -76,6 +77,7 @@
 					<a href="#" class="search">Search</a>
 					<a href="#" class="filter">Sort & Filter</a>
 					<a href="#" class="refresh">Refresh</a>
+					<a href="#" class="qr">QR Scanner</a>
```

Comment:
Add a condition to only show this when the QR code scanner is enabled (or use a CSS class, switched by JS, to hide it - but better not render it at all).

Reply:
Ah, so if the setting is set to "On", the camera would always show, and with "Off", only when accessed thought the menu?